stan/math/prim/scal/prob/wiener_log.hpp

```diff
@@ -132,6 +133,20 @@ namespace stan {
       VectorView<const T_tau> tau_vec(tau);
       VectorView<const T_delta> delta_vec(delta);
 
+      size_t N_y_tau = max_size(y, tau);
+      for (size_t i = 0; i < N_y_tau; i++)
+        if (y_vec[i] <= tau_vec[i]) {
+          std::stringstream msg;
+          msg << ", but must be greater than"
+                 "nondecision time which is ";
+          msg << tau_vec[i];
+          std::string msg_str(msg.str());
+          domain_error(function, "Random variable", y_vec[i],
+                       "is ", msg_str.c_str());
+          throw std::domain_error(msg_str.c_str());
```

Comment:
You can avoid including `<string>` and avoid a copy here (not that efficiency matters when about to throw an exception), and just use `msg.str().c_str()` directly.